src/main.js

```diff
@@ -33,6 +33,11 @@ app.setPath('userData', userDataPath);
 // Update cwd based on environment and platform
 setCurrentWorkingDirectory();
 
+// Register custom schemes with privileges
+protocol.registerSchemesAsPrivileged([
```

Comment:
@deepak1556 I suggest to follow up with @mjbvz , it seems to me we are using different options on master:

```ts
require('electron').webFrame.registerURLSchemeAsPrivileged('vscode-resource', {
    secure: true,
    bypassCSP: false,
    allowServiceWorkers: false,
    supportFetchAPI: true,
    corsEnabled: true
});
```

Also, since this happens on startup we got to be 100% sure this does not slow down our startup. Is it really necessary to do so early?